app/serializers/book_serializer.rb

```diff
@@ -0,0 +1,4 @@
+class BookSerializer < ActiveModel::Serializer
+  attributes :id, :gender, :author, :image, :title, :publisher, :year
+  has_many :rents
```

Comment:
As you are  sending the rents in the serializer, this is  making N+1 queries to the db, some info [here](https://semaphoreci.com/blog/2017/08/09/faster-rails-eliminating-n-plus-one-queries.html)

Reply:
Done